public/app/features/dashboard-scene/scene/types/LayoutRegistryItem.ts

```diff
@@ -17,4 +17,9 @@ export interface LayoutRegistryItem<S = {}> extends RegistryItem {
    * @param saveModel
    */
   createFromSaveModel?(saveModel: S): void;
+
+  /**
+   * Serializer for the layout
+   */
+  getSerializer(): LayoutManagerSerializer;
```

Comment:
Not sure this is the best place to stick the serializer. This ends up in the layoutManager.descriptor